src/uwtools/cli.py

```diff
@@ -479,7 +481,7 @@ def _add_arg_config_file(group: Group) -> None:
 
 def _add_arg_cycle(group: Group) -> None:
     group.add_argument(
-        "--cycle",
+        _switch(STR.cycle),
```

Comment:
A string literal slipped in here -- relocated to `STR`.

Reply:
😬 My bad!